src/029/p029.exs

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env elixir
+defmodule Problem029 do
+  defp generate_powers() do
+    for a <- 2..100,
+        b <- 2..100,
+        do: trunc(:math.pow(a, b))
+  end
+
+  def solve do
+    generate_powers()
+    |> Enum.to_list()
+    |> Enum.sort()
+    |> Enum.dedup()

```

Comment:
Hey, @fredericojordan!
I think you can use [Enum.uniq](https://hexdocs.pm/elixir/Enum.html#uniq/1), rather than `sort() |> dedup()`.
Can you try it?

Reply:
Done!
`|> Enum.to_list()` was also made superfluous, so I removed it.
Thanks! 😄